index.html

```diff
@@ -15,31 +15,294 @@
+                    <div class="Languages">
+                        <div class="language">Ruby on rails</div>
+                        <div class="language">Css</div>
+                        <div class="language">JavaScript</div>
+                        <div class="language">Html</div>
+                    </div>
+                    <button class="Read-more">See Project</button>
+
+                    <div class="cards">
+                        <img src="/Images/Img Plaholder.png" alt="Store image">
+                        <h4 class="title">Multi-Post Stories Gain+Glory</h1>
+                            <div class="Languages">
+                                <div class="language">Ruby on rails</div>
+                                <div class="language">Css</div>
+                                <div class="language">JavaScript</div>
+                                <div class="language">Html</div>
+                            </div>
+                            <button class="aboutme-button">See Project</button>
+                    </div>
+
+                    <div class="cards">
+                        <img src="/Images/Img Plaholder.png" alt="Store image">
+                        <h4 class="title">Multi-Post Stories Gain+Glory</h1>
+                            <div class="Languages">
+                                <div class="language">Ruby on rails</div>
+                                <div class="language">Css</div>
+                                <div class="language">JavaScript</div>
+                                <div class="language">Html</div>
+                            </div>
+                            <button class="Read-more">See Project</button>
+                    </div>
+                    <div class="cards">
+                        <img src="/Images/Img Plaholder.png" alt="Store image">
+                        <h4 class="title">Multi-Post Stories Gain+Glory</h1>
+                            <div class="Languages">
+                                <div class="language">Ruby on rails</div>
+                                <div class="language">Css</div>
+                                <div class="language">JavaScript</div>
+                                <div class="language">Html</div>
+                            </div>
+                            <button class="Read-more">See Project</button>
+                    </div>
+
+        </section>
+        <section class="About-me">
+            <div class="aboutme-text">
+                <div class="about-title">
+                    <h1>About me</h1>
+                </div>
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, </span>
+                    <br /> don’t hestiate to contact me.
+                </p>
+                <button class="aboutme-button">Get My Resume</button>
```

Comment:
- [ ] I kindly suggest that you add the different button interactions including hover, active (when it's pressed), disabled, and enable (default state) as shown in the Figma template below:

![image](https://user-images.githubusercontent.com/52934128/219883731-3d32d6f5-53ac-4128-a0c8-1a7595de03e5.png)

the button on the pressed state has the `background: #008552;` and on hover has the `background: #36B37F;`

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #008552;
}
```
**for hover state**
```
button:hover {
    background-color: #36B37F;
}
```